MobileBuy/buy/src/main/java/com/shopify/buy/utils/AndroidPayHelper.java

```diff
@@ -220,65 +224,39 @@ public static Address createShopifyAddress(UserAddress walletUserAddress) {
     }
 
     /**
-     * Creates a Masked Wallet Request from a Shopify Checkout
+     * Creates a Masked Wallet Request from a Shopify Checkout.
      *
      * @param merchantName        The merchant name to show on the Android Pay dialogs, not null or empty
      * @param checkout            The {@link Checkout} to use, not null.
      * @param publicKey           The Public Key to use, not empty.
      * @param phoneNumberRequired If true, the phone number will be required as part of the Shipping Address in Android Pay
      * @return A {@link MaskedWalletRequest}
      *
-     *
-     * @deprecated use {@link AndroidPayHelper#createMaskedWalletRequest(Checkout, Shop, String, boolean)} instead.
+     * @deprecated Use {@link AndroidPayHelper#createFullWalletRequest(Checkout, MaskedWallet)}
```

Comment:
son of a @#$@#@#$, guess it is time to go home for the day and try again tomorrow :)